config/routes.rb

```diff
@@ -1,19 +1,19 @@
+
 Rails.application.routes.draw do
   # The priority is based upon order of creation: first created -> highest priority.
   # See how all your routes lay out with "rake routes".
 
   # You can have the root of your site routed with "root"
-   root 'welcome#index'
-
+   #root 'welcome#index'
```

Comment:
Incorrect indentation detected (column 3 instead of 2).<br>Missing space after #.